TestFoundation/TestProcess.swift

```diff
@@ -7,6 +7,11 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+@testable import Foundation
```

Comment:
Same.

Since tests with `@testable import` only run for `--debug-foundation`, I’d strongly prefer that we instead find a way to bring in the functionality we need without invoking internal calls.